mr/src/main/java/org/apache/iceberg/mr/mapred/TableResolver.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.mr.mapred;
```

Comment:
Seems like this class is shared by both mrv2 and mrv1 apis, similar to InputFormatConfig. We should have a consistent package name there..

Reply:
In this PR it's only being used by the mrv1 API which is why it's in the `mapred` package. When we look at #933 after this we can discuss how much code is shared and then potentially move this up a package level and have it contain shared code etc.